catalog-info.yaml

```diff
@@ -63,3 +63,51 @@ spec:
           access_level: MANAGE_BUILD_AND_READ
         everyone:
           access_level: READ_ONLY
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-package-spec
+  description: 'EPR package specifications'
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/package-spec
+
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: package-spec
+      description: 'EPR package specifications'
+    spec:
+      branch_configuration: main
+      pipeline_file: ".buildkite/pipeline.yml"
+      provider_settings:
+        build_pull_request_forks: false
+        build_pull_requests: true
+        filter_enabled: true
+        filter_condition: >-
+          build.pull_request.id == null || (build.creator.name == 'elasticmachine' && build.pull_request.id != null)
+      repository: elastic/package-spec
+      cancel_intermediate_builds: true
+      cancel_intermediate_builds_branch_filter: '!main'
+      skip_intermediate_builds: true
+      skip_intermediate_builds_branch_filter: '!main'
+      env:
+        ELASTIC_SLACK_NOTIFICATIONS_ENABLED: 'true'
```

Comment:
Not sure about adding this env. var right now. The pipeline is not ready yet and it could cause some confusions to the authors of the PRs.
I would add it in a following PR once the pipeline is ready, or at least it has some steps working for tests.

Reply:
fixed